client/app/scripts/liveblog-edit/module.js

```diff
@@ -135,6 +137,11 @@ define([
                 $scope.panelState = panel;
                 // update url for deeplinking
                 $route.updateParams({panel: $scope.panelState});
+                //clear the new contribution notification
+                if (panel === 'contributions') {
+                    postsService.resetUnreadContributtions(panel);
+                }
+                postsService.setPanelState(panel);
```

Comment:
I don't understand which instruction is redundant with '$scope.panelState = panel; L137,'
The service should be aware of the panels because the number of unread contributions should only be increased if a panel other than the contributions one is active